src/Akka.Persistence.RavenDB/Query/RavenDbReadJournal.cs

```diff
@@ -27,16 +27,25 @@ public class RavenDbReadJournal ://TODO why called Journal if namespace is Query
         public readonly RavenDbPersistence Storage;
         private readonly Akka.Serialization.Serialization _serialization;
         
-        public RavenDbReadJournal(ExtendedActorSystem system, Config config)
+        private readonly RavenDbStore _store;
+        private readonly IActorRef _journalRef;
+        public RavenDbStore Store => _store;
+
+        public RavenDbReadJournal(ExtendedActorSystem system, Config queryConfig)//TODO stav: need to use this config to fetch query configuration (this contains query only)
         {
+            _journalRef = Persistence.Instance.Apply(system).JournalFor("");
```

Comment:
`.JournalFor("")` ?

Reply:
If it's empty it takes the default one. Once we implement the `_writeJournalPluginId` it should be added there